sdk/nodejs/cmd/run-policy-pack/run.ts

```diff
@@ -134,7 +134,7 @@ function throwOrPrintModuleLoadError(program: string, error: Error): void {
         }
     }
 
-    console.error("  * Yowzas, our sincere apologies, we haven't seen this before!");
+    console.error("  * We've encountered an unexpected error.");
     console.error(`    Here is the raw exception message we received: ${error.message}`);
```

Comment:
```suggestion
    console.error("  * Unexpected error.");
    console.error(`    Raw exception message: ${error.message}`);
```

😬 thank you for fixing this up! i wonder if `we` is the right thing to use here. don't feel strongly though so only use if its useful to you.

Reply:
https://github.com/pulumi/pulumi/pull/10370 is also for this change, and is already approved and good to go.